    try
    {
        // business logic
    }
    catch (Exception ex)
    {
        log.Error("Exception in MyMethodName()", ex);
    }
